<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DroneCompetitionClass</class>
 <widget class="QMainWindow" name="DroneCompetitionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DroneCompetition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_PaintArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>446</width>
      <height>290</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_FrontImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string>Front Image RGB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>54</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string>Front RGB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FrontImgRGB_Cap">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_FrontImageDep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string>Front Image Depth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>54</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string>Front DEP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FrontImgDep_Cap">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_DownImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string>Down Image RGB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>54</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string>Down RGB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DownImgRGB_Cap">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Test2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="gyo_z">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mag_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>acc_y</string>
    </property>
   </widget>
   <widget class="QLabel" name="mag_z">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>mag_y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS E</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Test3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="gps_e">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="gyo_y">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Test1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mag_x">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>gyo_x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>acc_x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>gyo_y</string>
    </property>
   </widget>
   <widget class="QLabel" name="gyo_x">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_z">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>acc_z</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_x">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="gps_n">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>gyo_z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>mag_z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>mag_x</string>
    </property>
   </widget>
   <widget class="QLabel" name="gps_h">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS h</string>
    </property>
   </widget>
   <widget class="QLabel" name="baro">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>BARO</string>
    </property>
   </widget>
   <widget class="QLabel" name="pz">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>Z_EST</string>
    </property>
   </widget>
   <widget class="QLabel" name="py">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>X_EST</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>Y_EST</string>
    </property>
   </widget>
   <widget class="QLabel" name="px">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="vz">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="vy">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="vx">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>54</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DroneCompetition.qrc"/>
 </resources>
 <connections/>
</ui>
